src/comparator.js

```diff
@@ -161,7 +178,9 @@ function fetchCSV (path: S3Path):Promise<customersMap> {
   console.log('Initialising parser.')
   return new Promise((resolve, reject) => {
     let line = 0
-    let reader = csv.parse({headers: true}).on('data', (data:customer) => {
+    let reader = csv.parse({headers: true}).on('data-invalid', function (data) {
+      console.log('ignoring invalid data: ' + data)
```

Comment:
I'm not %100 sure we can log this data here in case there is private information. I think we are already doing something similar in the exporter, we should confirm that it's ok to do this at some point

Reply:
we're fine to log to cloudwatch as it's encrypted